config.go

```diff
@@ -2,35 +2,57 @@ package main
 
 import (
 	"flag"
+	"os"
 
 	"github.com/vharitonsky/iniflags"
 )
 
 var (
+	VERSION    = "1.6.0"
 	appVersion = "unknown"
 	buildTime  = "unknown"
 )
 
 var (
-	logFile           = flag.String("logfile", "/var/log/smtprelay.log", "Path to logfile")
-	hostName          = flag.String("hostname", "localhost.localdomain", "Server hostname")
-	welcomeMsg        = flag.String("welcome_msg", "", "Welcome message for SMTP session")
-	listen            = flag.String("listen", "127.0.0.1:25 [::1]:25", "Address and port to listen for incoming SMTP")
-	localCert         = flag.String("local_cert", "", "SSL certificate for STARTTLS/TLS")
-	localKey          = flag.String("local_key", "", "SSL private key for STARTTLS/TLS")
-	localForceTLS     = flag.Bool("local_forcetls", false, "Force STARTTLS (needs local_cert and local_key)")
-	allowedNets       = flag.String("allowed_nets", "127.0.0.1/8 ::1/128", "Networks allowed to send mails")
-	allowedSender     = flag.String("allowed_sender", "", "Regular expression for valid FROM EMail addresses")
+	logFile       = flag.String("logfile", "/dev/stdout", "Path to logfile")
```

Comment:
Rather than use `/dev/stdout`, which [isn't necessarily portable](https://unix.stackexchange.com/q/36403/8928), why not just interpret an empty string to mean stdout? This would require just a little more work in `setupLogger()` (to skip the `os.OpenFile()` and just tell logrus to use stdout).

Reply:
Not fan of an empty value defaulting to `/dev/stdout` for the same reason.
I think we can revert back to the dedicated log file and folks can override as they see fit.

Reply:
The problem is one we had before already. There was some bad sed magic rewriting the config file for windows builds in the old release script. I nuked this with the migration to GitHub Action for release builds.

For the moment I would like to leave it as it is.